structuralcodes/geometry/_circular.py

```diff
@@ -0,0 +1,173 @@
+"""Classes for circular geometries.
+
+The class `CircularGeometry` represents a circular SurfaceGeometry with
+homogenous material.
+This class is simply a wrapper of `SurfaceGeometry` class and permits an easy
+input by the user.
+"""
+
+import typing as t
+from itertools import chain
+
+import numpy as np
+from shapely import Polygon
+
+from structuralcodes.core.base import ConstitutiveLaw, Material
+
+from ._geometry import CompoundGeometry, SurfaceGeometry
+from ._reinforcement import add_reinforcement_circle
+
+
+def _create_circle(radius, npoints=20):
+    """Create a circle with a given radius."""
+    phi = np.linspace(0, 2 * np.pi, npoints + 1)
+    x = radius * np.cos(phi)
+    y = radius * np.sin(phi)
+    points = [(this_x, this_y) for this_x, this_y in zip(x, y)]
```

Comment:
I suggest that we replace this with:

```python
points = np.transpose(np.array([x, y]))
```

It is about six times faster 😃

Reply:
I did not test it! but wow! for sure I will do it!

Reply:
Done in c29ed265e72637199db8ae79fae837c65cc47eca